### PR TITLE
Cloud update

### PR DESCRIPTION
**Pull request by msudberg**

### CODE REVIEW

File: doc/user/content/cloud/quickstart.md [hunk 1]
```diff
@@ -34,17 +34,21 @@ By default, you can create up to two deployments. If you're interested in more,
...
+For this example, we'll walk you through connecting to a [PubNub stream](https://www.pubnub.com/developers/realtime-data-streams/) as a data source. Note that PubNub demo streams should only be used for testing, since they are [volatile sources](/overview/volatility) that do not meet the consistency and durability requirements necessary for Materialize to guarantee correctness over time.
 
-1. From your shell, create a source (connect to the PubNub market orders stream):
+1. If you don't already have a PubNub account, [create one](https://dashboard.pubnub.com/signup) to obtain a subscribe key. (This is free.)
+
+2. From your shell, create a source (connect to the PubNub market orders stream):
 
     ```sql
     CREATE SOURCE market_orders_raw FROM PUBNUB
-      SUBSCRIBE KEY 'sub-c-4377ab04-f100-11e3-bffd-02ee2ddab7fe'
```
Comment (awang) [inline doc/user/content/cloud/quickstart.md:43]: Is it necessary to force users to get their own subscribe key?  This key is publicly available here: https://www.pubnub.com/developers/realtime-data-streams/financial-securities-market-orders/

It's a nice user experience to be able to copy-paste the snippet and not have to sign up for a new account

**Review by awang — COMMENTED** [review]
i'd strongly prefer to leave the subscribe key if possible.  the rest lgtm!

File: doc/user/content/cloud/quickstart.md (hunk 1 above)
Comment (msudberg) [reply doc/user/content/cloud/quickstart.md:43]: The one in the Developer doc is Petros' personal key, and we can use that or mine. They're probably fine for a small number of users, but they're rate-limited and may run into problems when we have more.

Comment (awang) [reply doc/user/content/cloud/quickstart.md:43]: I'm pretty sure it's not Petros' personal key!  It's listed on PubNub's website for everyone to use

**Review by awang — APPROVED** [review]
thank you!

**Review by ruf-io — APPROVED** [review]
All looks good

**Pull request merged**